studio/components/interfaces/TableGridEditor/SidePanelEditor/TableEditor/Column.tsx

```diff
@@ -150,7 +149,11 @@ const Column: FC<Props> = ({
             className="table-editor-column-type lg:gap-0 "
             disabled={column.foreignKey !== undefined}
             onOptionSelect={(format: string) => {
-              onUpdateColumn({ format, defaultValue: null })
+              let defaultValue = null
```

Comment:
let's keep things immutable too
```suggestion
             const defaultValue = format === 'uuid' ? 'gen_random_uuid()' : null
```

Reply:
Fixed.